src/components/CommunityStatus.tsx

```diff
@@ -0,0 +1,225 @@
+import i18n from 'assets/i18n';
+import BigNumber from 'bignumber.js';
+import { amountToCurrency } from 'helpers/currency';
+import {
+    calculateCommunityProgress,
+    calculateCommunityRemainedFunds,
+} from 'helpers/index';
+import { CommunityAttributes } from 'helpers/types/models';
+import { IRootState } from 'helpers/types/state';
+import React, { useEffect, useState } from 'react';
+import { View, StyleSheet, Dimensions } from 'react-native';
+import { Title, ProgressBar, Text, Divider } from 'react-native-paper';
+import { useSelector } from 'react-redux';
+import { ipctColors } from 'styles/index';
+
+import SuspiciousCard from './SuspiciousCard';
+import Card from './core/Card';
+import WarningRedTriangle from './svg/WarningRedTriangle';
+
+interface ICommuntyStatusProps {
+    children?: any; // linter issues are a bit anoying sometimes
+    community: CommunityAttributes;
+}
+
+export default function CommunityStatus(props: ICommuntyStatusProps) {
+    const { community } = props;
+    const { width } = Dimensions.get('screen');
+    const [communityGoal, setCommunityGoal] = useState<BigNumber>();
+    const [remainedFunds, setRemainedFunds] = useState<number>();
+    const user = useSelector((state: IRootState) => state.user.metadata);
+
+    const app = useSelector((state: IRootState) => state.app);
```

Comment:
we are using exhangeRates too. I didn't get the point of the comment here.

Reply:
What I mean is that I've only found usage to `app.exchangeRates`.
So it could be `const exchangeRates = useSelector((state: IRootState) => state.app.exchangeRates);` instead

Reply:
Roger that now. Fixed.